repos/system_upgrade/el7toel8/actors/sctpconfigread/actor.py

```diff
@@ -0,0 +1,71 @@
+from leapp.actors import Actor
+from leapp.models import SCTPConfig, SystemFacts
+from leapp.tags import FactsPhaseTag, IPUWorkflowTag
+
+
+class SCTPConfigRead(Actor):
+    name = 'sctp_read_status'
+    description = 'This actor reads SCTP status, if it was/is used.'
+    consumes = (SystemFacts,)
```

Comment:
Please use `ActiveKernelModulesFacts` instead of `SystemFacts` now. We have split the SystemFacts model into separate models. Sorry I did miss that before. The CI just showed me that on the failure.